rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -166,16 +166,9 @@ NodeGraph::get_node_names() const
 size_t
 NodeGraph::count_publishers(const std::string & topic_name) const
 {
-  auto rmw_node_handle = rcl_node_get_rmw_handle(node_base_->get_rcl_node_handle());
-  auto fqdn = rclcpp::expand_topic_or_service_name(
```

Comment:
This step will combine the topic name and namespace into a "fully qualified" topic name [(example)](https://github.com/ros2/rclcpp/blob/b90676871d397cbfdcfecdf265869efb0bd97d9f/rclcpp/test/test_expand_topic_or_service_name.cpp#L26) and should not be removed